docs/api/app.md

```diff
@@ -330,3 +330,13 @@ Sets the application's [dock menu][dock-menu].
 
 [dock-menu]:https://developer.apple.com/library/mac/documentation/Carbon/Conceptual/customizing_docktile/concepts/dockconcepts.html#//apple_ref/doc/uid/TP30000986-CH2-TPXREF103
 [tasks]:http://msdn.microsoft.com/en-us/library/windows/desktop/dd378460(v=vs.85).aspx#tasks
+
+## `app.defaultSession`
```

Comment:
I would like to keep it undocumented for now, since the design of session API is not decided yet.